style/index.css

```diff
@@ -36,7 +38,7 @@
   flex-direction: column;
   background: var(--jp-layout-color1);
   color: var(--jp-ui-font-color1);
-  font-size: var(--jp-ui-font-size0);
+  /* font-size: var(--jp-ui-font-size0);  */
```

Comment:
I still think this should be restored: the headers in the other side panels have slightly smaller text, and this makes it more consistent with those.